challenges/prototypes.js

```diff
@@ -5,21 +5,27 @@
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
-
+function CuboidMaker(props) {
+  this.length = props.length;
+  this.width = props.width;
+  this.height = props.height;
+}
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
-  
+
   Formula for cuboid volume: length * width * height
 */
-
+CuboidMaker.prototype.volume = function() {
+  return volume;
```

Comment:
Volume is not defined here - this function should calculate the volume based on the formula given above.